### PR TITLE
DOC: Convert CompareFlights example notebooks to .rst files

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

- [x] ReadMe, Docs and GitHub updates

## Checklist

- [ ] Tests for the changes have been added (not needed)
- [x] Docs have been reviewed and added / updated
- [x] Lint (`black rocketpy/ tests/`) has passed locally 
- [ ] All tests (`pytest tests -m slow --runslow`) have passed locally
- [ ] `CHANGELOG.md` has been updated (if relevant)

## Current behavior
The example notebooks are not executed when building the documentation. 
There's no way to add custom widgets (i.e. "seealso", "important", etc.)

## New behavior
Two .ipynb files were converted to .rst files.
Sphinx will now execute the code cells whenever we build the docs.
- It prevents our documentation to stay outdated.
- This increases building time a little, but the result is worth it imo.

## Breaking change

- [x] No

## Additional information
- Please check the read the docs, if possible.

### CODE REVIEW

File: docs/user/compare_flights.rst [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+              flights.append(flight)
+
+
+We can easily visualize the number of flights created:
+
+.. jupyter-execute::
+
+      print("Number of flights: ", len(flights))
+
+Start the comparison
```
Comment (MateusStano) [inline docs/user/compare_flights.rst:133]: Might be useful to add a `see also` with a link to the code reference page of the class here and/or at the beginning of the document

File: docs/user/compare_flights.rst [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+We will start by importing the necessary classes and modules:
+
+.. jupyter-execute::
+
+      from rocketpy.plots.compare import CompareFlights
+      from rocketpy import Environment, Flight, Rocket, SolidMotor
+      from datetime import datetime, timedelta
+
+
+Create Environment, Motor and Rocket
```
Comment (MateusStano) [inline docs/user/compare_flights.rst:21]: A `see also` to the first simulation page would be useful here

File: docs/user/deployable.rst [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+          throat_radius=11 / 1000,
+          coordinate_system_orientation="nozzle_to_combustion_chamber",
+      )
+
+.. jupyter-execute::
+
+      Pro75M1670.info()
+
+Simulating the 1st Flight: ascending phase
+------------------------------------------
```
Comment (MateusStano) [inline docs/user/deployable.rst:78]: Maybe a see also to the first simulation page would be useful here

File: docs/user/compare_flights.rst (hunk 2 above)
Comment (Gui-FernandesBR) [reply docs/user/compare_flights.rst:21]: done [ca74af0](https://github.com/RocketPy-Team/RocketPy/pull/576/commits/ca74af040125039dcada325d740892fc83cd21a4)

File: docs/user/compare_flights.rst (hunk 1 above)
Comment (Gui-FernandesBR) [reply docs/user/compare_flights.rst:134]: done [ca74af0](https://github.com/RocketPy-Team/RocketPy/pull/576/commits/ca74af040125039dcada325d740892fc83cd21a4)

**Pull request merged**